model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -92,22 +92,25 @@ class ExcludesTest : WordSpec() {
             "find the correct path excludes for a path" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
-                excludes.findPathExcludes("") should beEmpty()
-                excludes.findPathExcludes("path1") should containExactly(pathExclude1)
-                excludes.findPathExcludes("path2") should containExactly(pathExclude2)
-                excludes.findPathExcludes("test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/file.ext") should containExactly(pathExclude3, pathExclude4)
+                excludes.apply {
```

Comment:
I believe we *mostly* use `with()` to group assertions in tests, also see [apply {} vs with()](https://stackoverflow.com/a/36625603/1127485), and I have a *slight* tendency towards `with()` as it arguably reads a bit nicer in this context, but I'm not blocking this change. Any opinions?

Reply:
I also find it slightly nicer and it's more consistent, so let's use it.